lib/rules/jsx-quotes.js

```diff
@@ -19,11 +21,19 @@ var astUtils = require("../ast-utils");
 var QUOTE_SETTINGS = {
     "prefer-double": {
         quote: "\"",
-        description: "singlequote"
+        alternateQuote: "'",
+        description: "singlequote",
+        convert: function(str) {
+            return toDoubleQuotes(str);
```

Comment:
+1 to that. HTML escaping is different, please add some tests with embedded quotes.